tests/drivers/disk/prj.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_TEST=y
+CONFIG_ZTEST=y
+CONFIG_DISK_ACCESS=y
```

Comment:
`+ CONFIG_DISK_DRIVER_SDMMC=y`

Reply:
Should `CONFIG_DISK_DRIVER_SDMMC` be selected unconditionally, even when testing a RAM disk?

Reply:
I think one should be enabled per default, you PR is targeting SDMMC driver, I can add test case for ramdisk in #40611

Reply:
Sounds good. `CONFIG_DISK_DRIVER_SDMMC` should be selected for the test in the latest push